botocore/client.py

```diff
@@ -522,7 +530,7 @@ def _service_model(self):
         return self.meta.service_model
 
     def _make_api_call(self, operation_name, api_params):
-        request_context = {}
+        request_context = {'client_region': self.meta.region_name}
```

Comment:
It would be nice to have consistent casing for keys in this dict.  Looks like camel case is being used for the encoding type key.